drivers/storage/portworx/portworx.go

```diff
@@ -140,6 +156,12 @@ func (p *portworx) Validate(cluster *corev1.StorageCluster) error {
 		return nil
 	}
 
+	if err := EnableSecurityContextForValidate(p.recorder, cluster); err != nil {
```

Comment:
Can we rename this to `reconcileSecurity...` or `createSecurity...`? We are not enabling the SCCs, but we are just creating them if they don't exist. 
Also, change the function name to lowerCamelCase as it doesn't need to be public.

Reply:
oh shit sorry... yeah